src/utils/test-utils/src/odf.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub fn alias(account: &str, dataset_name: &str) -> odf::DatasetAlias {
+    odf::DatasetAlias::new(
+        Some(odf::AccountName::new_unchecked(account)),
+        odf::DatasetName::new_unchecked(dataset_name),
+    )
+}
```

Comment:
Why not to put it to `odf/testing` itself?

Reply:
Indeed, the proposed location is better

Moved